pkg/dogstatsd/server.go

```diff
@@ -97,6 +86,11 @@ func NewServer(metricOut chan<- *metrics.MetricSample, eventOut chan<- metrics.E
 		metricPrefix = metricPrefix + "."
 	}
 
+	defaultHostname, err := util.GetHostname()
+	if err != nil {
+		log.Error(err.Error())
```

Comment:
We might want to make this a little more descriptive as to the impact of the `error`.